src/personen/personen.lambda.ts

```diff
@@ -0,0 +1,107 @@
+import * as https from 'https';
+import { AWS } from '@gemeentenijmegen/utils';
+import { DynamoDB } from 'aws-sdk';
+import nodefetch from 'node-fetch';
+
+export async function handler (event: any, _context: any):Promise<any> {
+  const request = JSON.parse(event.body);
+  const apiKey = event.requestContext.identity.apiKey;
+
+  const idTable = new DynamoDB.DocumentClient();
+
+  const validProfile = await validateFields(request.fields, apiKey, idTable);
+
+  if (validProfile) {
+    // Search...
+    return callHaalCentraal(event.body, await getSecrets());
```

Comment:
Dit haat voor elk request de secrets opnieuw op. Ik zou ze cachen in de lambda, als ze dan al geladen zijn return je die vanuit de GetSecrets functie anders haal je ze eerst op.